File: leet_0389_find_char___fastest.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        res = ord(t[-1])
        i = 0
        while i < len(s):
            res += ord(t[i])
            res -= ord(s[i])
            i += 1
        return chr(res)

File: leet_0389_find_char___fastest.cc
class Solution {
public:
    char findTheDifference(string s, string t)
    {
        int res = t[t.length() - 1];
        int i = -1;
        while (++i < s.length())
        {
            res += t[i];
            res -= s[i];
        }
        return (char) res ;
    }
};
